Development/nmos/test/query_api_test.cpp

```diff
@@ -20,27 +20,23 @@ BST_TEST_CASE(testQueryAPISubscriptionsExtensionSchema)
         boost::copy_range<std::vector<web::uri>>(nmos::is04_versions::all | boost::adaptors::transformed(nmos::experimental::make_queryapi_subscriptions_post_request_schema_uri))
     };
 
-    // valid subscriptions data
+    // valid subscriptions post request data
+    // see https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.3.x/examples/queryapi-subscriptions-post-request.json
     auto data = value_of({
         { U("max_update_rate_ms"), 100 },
         { U("resource_path"), U("/nodes") },
         { U("params"), value_of({
             { U("label"), U("host1") }
         }) },
         { U("persist"), false },
-        { U("secure"), false },
-        { U("authorization"), false },
-        { U("ws_href"), U("ws://172.29.80.52:8870/ws/?uid=7c903667-7113-4a8f-8865-1c63f9070a9e") },
-        { U("id"), U("7c903667-7113-4a8f-8865-1c63f9070a9e") }
+        { U("secure"), false }
```

Comment:
The `secure` field is not required by the schema, we can drop that too.

Reply:
You could, but the example in the spec does include that property. 🤷‍♂️